packages/widget/src/assets/icons/chevron.ts

```diff
@@ -6,6 +6,7 @@ const chevronIcon = html`<svg
   height="24"
   viewBox="0 0 24 24"
   fill="none"
+  part="chevronIcon"
```

Comment:
what is this for?

Reply:
The initial intention was to customize the size of the icon. If it renders for example as a part of the dropdown component, but you need a connect wallet dropdown with a custom size and color of the icon then "part" is one of the ways to access shadow dom. But I re-check the code and looks like I abandoned this idea 😃